addons/beehave/nodes/beehave_root.gd

```diff
@@ -7,6 +7,13 @@ const SUCCESS = 0
 const FAILURE = 1
 const RUNNING = 2
 
```

Comment:
Could you name this ProcessMode instead?

Reply:
Done!